bin/k8s-deploy

```diff
@@ -152,6 +152,24 @@ do
     echo "Updating ${DEPLOYMENT_FILE}-${CI_SHA1}"
     DEPLOYMENT_FILE=${DEPLOYMENT_FILE}-${CI_SHA1}
   fi
+  if [[ $( grep -i replicas $DEPLOYMENT_FILE | awk -F: {'print $2'} | grep -i hpa ) ]]
+  then
+    REPLICA_COUNT=$( kubectl get deployment $DEPLOYMENT --namespace=$NAMESPACE |
+                       awk -v subj="CURRENT" '{
+                         for(n=1;n<=NF;n++){
+                           if ($n==subj){
+                             subp=n;nrr=NR
+                           }
+                         };
+                         if(NR!=nrr){
+                           print $subp
+                         }
+                       }'
+                   )
```

Comment:
Is there a reason to use awk to parse the human-UI portion instead of `jq` and the json output? It looks like this can be done with:

```
kubectl get deployment $DEPLOYMENT --namespace=$NAMESPACE -o json | jq '.status.readyReplicas'
```
which (IMO) seems a bit simpler to read and more reliable than parsing CLI output.